src/conversion_helper.js

```diff
@@ -711,24 +721,30 @@ function findPotentialConversions(comment) {
 /*
   Input: Array of input numbers and standardized units
     [
-      { "imperial": { "number" : 10000, "unit" : " miles" } },
-      { "imperial": { "number" : -2, "unit" : " miles" } },
-      { "imperial": { "number" : 3, "unit" : " mph" } }
+      { "imperial": { "numbers" : 10000, "unit" : " miles" } },
+      { "imperial": { "numbers" : -2, "unit" : " miles" } },
+      { "imperial": { "numbers" : 3, "unit" : " mph" } }
     ]
   Output: Valid conversions
     [
-      { "imperial": { "number" : 10000, "unit" : " miles" } },
-      { "imperial": { "number" : 3, "unit" : " mph" } }
+      { "imperial": { "numbers" : 10000, "unit" : " miles" } },
+      { "imperial": { "numbers" : 3, "unit" : " mph" } }
     ]
 */
 function filterConversions(potentialConversions) {
   const possiblyValidConversions = potentialConversions.filter(input => {
     const imperialUnit = input['imperial']['unit'];
-    const imperialNumber = Number(input['imperial']['number']);
+    const imperialNumbers = input['imperial']['numbers'];
 
     const map = unitLookupMap[imperialUnit];
     if (map['isInvalidInput']) {
-      return map['isInvalidInput'](imperialNumber) == false;
+      let result = true;
+      imperialNumbers.forEach(function(item) {
```

Comment:
Can you change this to a "for-in" loop as well? I'm not a fan of the "result" variable when we can just return out of the function with a false